tests/rt.conf

```diff
@@ -1,325 +1,395 @@
-RUN     | cpld_restart_bmark_p8                                                                                                   | - s4.intel jet.intel cheyenne.intel acorn.intel  |     | cpld_bmark_p8
-
-COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                                         |                                         | fv3 |
-RUN     | cpld_control_noaero_p8                                                                                                  |                                         | fv3 |
-RUN     | cpld_control_nowave_noaero_p8                                                                                           |                                         | fv3 |
-
-COMPILE | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                       | - wcoss2.intel acorn.intel              | fv3 |
-RUN     | cpld_debug_p8                                                                                                           | - wcoss2.intel acorn.intel              | fv3 |
-
-COMPILE | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                              | - wcoss2.intel acorn.intel              | fv3 |
-RUN     | cpld_debug_noaero_p8                                                                                                    | - wcoss2.intel acorn.intel              | fv3 |
+### RT.CONF FORMATTING ###
+# COMPILE Line ( Items separated by a | )
+# Item 1: COMPILE - This tells rt.conf the following information is to be used in setting up a compile job
+# Item 2: Compile number - must be sequential in rt.conf - use as a reference for compile failures
+# Item 3: Compiler to use in build (intel or gnu)
+# Item 4: CMAKE Options - Provide all CMAKE options for the build
+# Item 5: Machines to run on (- is used to ignore specified machines, + is used to only run on specified machines) 
+## -> EX: + hera orion gaea = compile will only run on hera orion and gaea machines 
+## -> EX: - wcoss2 acorn = compile will NOT be run on wcoss2 or acorn
+# Item 6:
+#
+# RUN Line ( Items separated by a | )
+## NOTE: The build resulting from the COMPILE line above the RUN line will be used to run the test 
+# Item 1: RUN - This tells rt.conf the following information is to be used in setting up a model run
+# Item 2: Test name - Which test in the tests/tests directory should be sourced?
+# Item 3: Machines to run on (- is used to ignore specified machines, + is used to only run on specified machines) 
+## reference example above
+# Item 4: Controls whether the run creates its own baseline or it uses the baseline from a different (control) test
+
+### Intel Tests ###
+### S2S tests ###
+COMPILE | 1 | intel | -DAPP=S2SWA -D32BIT=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp | | fv3 |
```

Comment:
I did not feel it was important for COMPILE lines to match their following RUN lines columns because the data for each line is drastically different and creates an unnecessary large line of blank spaces.

The new changes bring what you mention to the RUN lines, they are much easier to follow.

I will leave this open for opinions, happy to make changes if I am in the minority! @junwang-noaa @jkbk2004 @DusanJovic-NOAA what do you feel?

Reply:
I didn't mean to comment on the compile line specifically (it was just the first line in the file). My comment was more about the alignment of | in the actual run lines.

Reply:
Thats fine. It's still a similar answer, though. I didn't feel the RUN lines needed to share the same shape as other RUN lines that followed different COMPILE lines, as the test names all vary in length, and the different machines to ignore or only run on varied as well.

Reply:
I personally prefer that all RUN lines are aligned. For example if I want to see the status of all restart tests I can grep for RUN lines with string restart and see :

```
$ grep "^RUN" rt.conf | grep restart
RUN | cpld_restart_p8       |                            |     | cpld_control_p8
RUN | cpld_restart_qr_p8    |                            |     | cpld_control_qr_p8
RUN | cpld_restart_c192_p8  | - wcoss2 jet acorn gaea s4 |     | cpld_control_c192_p8
RUN | cpld_restart_bmark_p8 | - s4 jet cheyenne acorn    |     | cpld_bmark_p8
RUN | cpld_restart_c48   | |     | cpld_warmstart_c48
RUN | cpld_restart_pdlib_p8 | + hera orion cheyenne  |     | cpld_control_pdlib_p8
RUN | control_stochy_restart           |                |     | control_stochy
RUN | control_restart_p8               |                |     | control_p8
RUN | control_restart_qr_p8            |                |     | control_qr_p8
RUN | regional_restart                 |                |     | regional_control
RUN | regional_restart_qr              |                |     | regional_control_qr
RUN | rap_restart                               | |     | rap_control
RUN | rap_sfcdiff_restart                       | |     | rap_sfcdiff
RUN | hrrr_control_restart                      | |     | hrrr_control
RUN | rrfs_conus13km_hrrr_warm_restart_mismatch | | fv3 | rrfs_conus13km_hrrr_warm
RUN | rap_restart_dyn32_phy32                 | |     | rap_control_dyn32_phy32
RUN | hrrr_control_restart_dyn32_phy32        | |     | hrrr_control_dyn32_phy32
RUN | datm_cdeps_restart_cfsr        | - wcoss2 |     | datm_cdeps_control_cfsr
RUN | rap_restart                             | + hera cheyenne |     | rap_control
RUN | rap_sfcdiff_restart                     | + hera cheyenne |     | rap_sfcdiff
RUN | hrrr_control_restart                    | + hera cheyenne |     | hrrr_control
RUN | rrfs_conus13km_hrrr_warm_restart_mismatch | + hera cheyenne | fv3 | rrfs_conus13km_hrrr_warm
RUN | rap_restart_dyn32_phy32           | + hera cheyenne |     | rap_control_dyn32_phy32
RUN | hrrr_control_restart_dyn32_phy32  | + hera cheyenne |     | hrrr_control_dyn32_phy32
```
Which is not as readable as, for example:

```
$ grep "^RUN" rt_aligned.conf | grep restart
RUN | cpld_restart_p8                           |                            |     | cpld_control_p8
RUN | cpld_restart_qr_p8                        |                            |     | cpld_control_qr_p8
RUN | cpld_restart_c192_p8                      | - wcoss2 jet acorn gaea s4 |     | cpld_control_c192_p8
RUN | cpld_restart_bmark_p8                     | - s4 jet cheyenne acorn    |     | cpld_bmark_p8
RUN | cpld_restart_c48                          |                            |     | cpld_warmstart_c48
RUN | cpld_restart_pdlib_p8                     | + hera orion cheyenne      |     | cpld_control_pdlib_p8
RUN | control_stochy_restart                    |                            |     | control_stochy
RUN | control_restart_p8                        |                            |     | control_p8
RUN | control_restart_qr_p8                     |                            |     | control_qr_p8
RUN | regional_restart                          |                            |     | regional_control
RUN | regional_restart_qr                       |                            |     | regional_control_qr
RUN | rap_restart                               |                            |     | rap_control
RUN | rap_sfcdiff_restart                       |                            |     | rap_sfcdiff
RUN | hrrr_control_restart                      |                            |     | hrrr_control
RUN | rrfs_conus13km_hrrr_warm_restart_mismatch |                            | fv3 | rrfs_conus13km_hrrr_warm
RUN | rap_restart_dyn32_phy32                   |                            |     | rap_control_dyn32_phy32
RUN | hrrr_control_restart_dyn32_phy32          |                            |     | hrrr_control_dyn32_phy32
RUN | datm_cdeps_restart_cfsr                   | - wcoss2                   |     | datm_cdeps_control_cfsr
RUN | rap_restart                               | + hera cheyenne            |     | rap_control
RUN | rap_sfcdiff_restart                       | + hera cheyenne            |     | rap_sfcdiff
RUN | hrrr_control_restart                      | + hera cheyenne            |     | hrrr_control
RUN | rrfs_conus13km_hrrr_warm_restart_mismatch | + hera cheyenne            | fv3 | rrfs_conus13km_hrrr_warm
RUN | rap_restart_dyn32_phy32                   | + hera cheyenne            |     | rap_control_dyn32_phy32
RUN | hrrr_control_restart_dyn32_phy32          | + hera cheyenne            |     | hrrr_control_dyn32_phy32
```
where I can immediately see that there are two 'restart' tests that generate new baselines and I can reason why and whether that's intended or not. Or see the status of restart tests on various machines. Or I can verify that all restart tests have it's dependent test specified in the last column. Etc.

Reply:
Maybe this PR is good opportunity to finally replace that 'fv3' in the 4th column, that means 'generate new baselines' to some other string. 'baseline' or 'base' maybe? Or something else other than 'fv3'.

Reply:
i've changed them to baseline and addressed where i think it matters in rt.sh